llama/sap.py

```diff
@@ -402,3 +403,28 @@ def generate_sap_control(sap_data_file: str, invoice_total: float) -> str:
     sap_control_file += "\n"
 
     return sap_control_file
+
+
+def generate_next_sap_sequence_number() -> str:
+    """Get the current SAP sequence parameter from SSM and return only the sequence number
+    incremented by 1."""
+    ssm = SSM()
+    sap_sequence_parameter = ssm.get_parameter_value(CONFIG.SSM_PATH + "SAP_SEQUENCE")
+    split_parameter = sap_sequence_parameter.split(",")
+    return str(int(split_parameter[0]) + 1)
+
+
+def update_sap_sequence(
+    old_sap_sequence: str, date: datetime, sequence_type: str
+) -> str:
+    """Update SAP sequence and post it to SSM Parameter Store."""
```

Comment:
Well, looking at what I said in my comment and it was still wrong (facepalm). This function doesn't need the old sequence parameter at all, it only need the current sequence number (the one we want to update with). And then it can generate a sequence param with that, the formatted date, and the type.